Update environment.yml

ctr26 opened the pull request:
`/opt/conda/bin/python: No module named jupyter-notebook
`
Might fix the above error, unsure, untested.

ctr26 committed:
Update environment.yml

/opt/conda/bin/python: No module named jupyter-notebook

Might fix the above error, unsure, untested.

ctr26 commented:
`exec /opt/conda/bin/python -m "$@"`

This line might also be problematic maybe

`conda run  "$@"`

Would be better?

oeway commented:
Let's try it then: https://alpha.binder.bioimagearchive.org/v2/gh/ctr26/imjoy-hybrid-desktop/patch-1



For jupyter notebook, I have this line already though: https://github.com/imjoy-team/imjoy-hybrid-desktop/blob/e73f8cef5b75c5ab24433d0fa4ce71508f433024/binder/Dockerfile#L46

ctr26 committed:
Update start

Out of ideas otherwise

ctr26 commented:
> `exec /opt/conda/bin/python -m "$@"`
> 
> This line might also be problematic maybe
> 
> `conda run  "$@"`
> 
> Would be better?

I'll try this too

ctr26 committed:
Update start

 PermissionError: [Errno 13] Permission denied: '/opt/conda/.tmpu295rsg_'

`$ /opt/conda/bin/conda run jupyter-notebook --ip=0.0.0.0 --port=8888 --NotebookApp.base_url=/jhub/user/ctr26-imjoy-hybrid-desktop-1oia8yxn/ --NotebookApp.token=pOJS1L21SDm-4CjHyXcRkw --NotebookApp.trust_xheaders=True --NotebookApp.allow_origin=* --NotebookApp.allow_origin_pat=.*`

oeway committed:
RUN chown -R ${NB_USER} ${HOME}

oeway committed:
Update Dockerfile

ctr26 committed:
Permissions fix
